pkgs/development/python-modules/psycopg2cffi/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildPythonPackage, cffi, fetchFromGitHub, lib, postgresql, pytest, six }:
+
+buildPythonPackage rec {
+  pname = "psycopg2cffi";
+  version = "2.8.1";
+
+  src = fetchFromGitHub {
```

Comment:
maybe worth adding a comment that this is a fork

Reply:
Done!